src/contentscript/overlay/root/components/OverlayTab/index.tsx

```diff
@@ -52,54 +51,36 @@ export interface OverlayTabProps {
   connectedDescriptors?: any
   selectedWallet?: any
   isToolbar?: boolean
-  events?: any
 }
 
 export const OverlayTab = (p: OverlayTabProps): ReactElement => {
   const visibleMenus = p.menus.filter((x) => x.hidden !== true)
   const nodeVisibleMenu = useRef<HTMLDivElement>()
   const [menuVisible, setMenuVisible] = useState(false)
-  const [event, setEvent] = useState([])
+  const [event, setEvent] = useState<Notify[]>([])
+
+  useEffect(() => {
+    !document
+      .querySelector('#dapplets-overlay-manager')
+      .classList.contains('dapplets-overlay-collapsed') && setMenuVisible(false)
+    return () => {}
+  }, [menuVisible])
+
   useEffect(() => {
-    const init = async () => {
+    const handleUpdateNotifications = async () => {
       const notifications = await getNotifications()
       setEvent(notifications && notifications.filter((x) => x.status === 1))
```

Comment:
`x.status === 1` to `x.status === NotificationStatus.XXX`

Reply:
++